src/lib.rs

```diff
@@ -310,7 +310,10 @@ fn generate_symlink<P: AsRef<Path>, Q: AsRef<Path>>(symlink_path: P, output_path
     if symlink_path.as_ref().exists() {
         std::fs::remove_file(&symlink_path).map_err(Error::SymlinkCreationFailure)?;
     }
+    #[cfg(unix)]
     std::os::unix::fs::symlink(output_path, symlink_path).map_err(Error::SymlinkCreationFailure)?;
+    #[cfg(windows)]
+    junction::create(output_path, symlink_path).map_err(Error::SymlinkCreationFailure)?;
```

Comment:
Looks like the they're trying to add support on the standard library, but it's still nightly :/
https://github.com/rust-lang/rust/issues/121709

Reply:
Yea I have a requirement to run on rust 1.77 so I couldn't do that

Reply:
Yeah absolutely, I don't want to turn this into a nightly-only crate